yarn.lock

```diff
@@ -11996,7 +12022,7 @@ raw-loader@^3.1.0:
     loader-utils "^1.1.0"
     schema-utils "^2.0.1"
 
-react-dom@^17.0.1:
+react-dom@17.0.1, react-dom@^17.0.1:
```

Comment:
Hm, it looks like we still have some stray changes in here. Can we revert the `yarn.lock` file and re-run `yarn add -D eslint-plugin-import@^2.22.1`?